imagekit_test.go

```diff
@@ -262,45 +296,14 @@ func TestUrl(t *testing.T) {
 			if err != nil {
 				t.Errorf(err.Error())
 			}
-
-			if strings.Index(tc.url, "tr:") > -1 {
-				url, tr := extractTransformation(t, url)
-				expectedUrl, expectedTr := extractTransformation(t, tc.url)
-
-				if !urlsEquals(url, expectedUrl) {
-					t.Errorf("expected url: %s\ngot: %s", expectedUrl, url)
-				}
-
-				if !cmp.Equal(tr, expectedTr) {
-					t.Errorf("url: %s, expected tr: %s\ngot tr: %s", tc.url, expectedTr, tr)
-				}
+			if !urlsEquals(url, tc.url) {
```

Comment:
There was only an assertion for URLs containing `tr:` inside the path, so 
 here added an assertion that will test all the URLs.